File: post.pl
#!/usr/bin/perl -w 
use v5.18.2;


if ($#ARGV == 0 || lc($ARGV[0]) eq 'xiu') {
    say "🐬🐬🐬 Submint through account : XIU";
    `cp /Users/alpha/.lc/xiu.json /Users/alpha/.lc/leetcode/user.json `;
}  elsif ( $#ARGV >= 1 && lc($ARGV[0]) eq '117') {
    say "🐐🐐🐐 Submint through account : 117";
    `cp /Users/alpha/.lc/firefly.json /Users/alpha/.lc/leetcode/user.json `;
} else {
    die "Choices are either XIU or 117";
}

say `/usr/local/bin/proxychains4 -q /usr/local/bin/leetcode submit $ARGV[$#ARGV]`;

File: post.py
#!/usr/bin/env python3
from collections import Counter, defaultdict, OrderedDict, deque
from bisect import bisect_left, bisect_right 
from functools import reduce, lru_cache 
from typing import List 
import itertools 
import math 
import heapq 
import string
true = True
false = False
MIN, MAX = -0x3f3f3f3f, 0x3f3f3f3f
#!/usr/local/bin/python3
import sys 
import os  

if len(sys.argv) == 2 or sys.argv[1] == "xiu":
    print("🐬🐬🐬 Submint through account : XIU")
    os.system("cp /Users/alpha/.lc/xiu.json /Users/alpha/.lc/leetcode/user.json")
elif sys.argv[1] == '117':
    print("🔥🔥🔥 Submint through account : 117")
    os.system("cp /Users/alpha/.lc/firefly.json /Users/alpha/.lc/leetcode/user.json")
else:
    print("Either xiu or 117")
    exit(0)

if sys.argv[-1].endswith('rs'):
    print("Submitting Rust solution")
    f = open('question.rs', 'r').read().__str__().replace('pub struct Solution;', '// pub struct Solution;')
    with open(sys.argv[-1], 'w') as wh:
        wh.write(f)

os.system('/usr/local/bin/proxychains4 -q leetcode submit {}'.format(sys.argv[-1]))






